src/main/java/io/logz/guice/jersey/JerseyServer.java

```diff
@@ -76,4 +99,26 @@ protected Injector getInjector() {
         server.setHandler(webAppContext);
     }
 
+    private void validate(JettyConfiguration jettyConfiguration) {
```

Comment:
these check should be performed in the constrictor of the config object, not here.